.travis.yml

```diff
@@ -13,4 +13,4 @@ cache:
     - $HOME/.gradle/wrapper/
 install: true
 script:
-    - ./gradlew check --no-daemon
+    - ./gradlew :spring-rabbit:check --no-daemon -Dtest.single=DirectMessageListenerContainerTests --info
```

Comment:
Yes; it failed again - see here https://travis-ci.org/spring-projects/spring-amqp/builds/161924553 - but once again, with this here now it passes again - I fear we will have to wait until it fails on bamboo to debug it.